src/HydraEngine/engine/engine.ts

```
@@ -107,11 +107,15 @@ export default class Hydra {
         switch (component.type) {
           case "sprite":
             const spriteComponentData = component.data as ISpriteComponent;
+            let sprite: Sprite = null;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)